rosetta/rosetta/projects/pax/README.md

```diff
@@ -93,19 +93,26 @@ Paxml uses [Fiddle](https://github.com/google/fiddle/tree/main) for configuring
 For example, in our `*.sh` scripts, we override the default values of `FPROP_DTYPE`, `ICI_MESH_SHAPE`, and `PERCORE_BATCH_SIZE`. 
 
 We provide a list of some of the frequently overridden hyperparameters, and an explanation of each, below:
-- `ICI_MESH_SHAPE`: This refers to the parallelism strategy used on chips connected by a fast network (e.g. NVLink). `ICI_MESH_SHAPE` typically has 3 dimensions, `[data, fsdp, tensor]`, corresponding to data parallelism (DP), fully-sharded data parallelism (FSDP/ZeRO-3), and tensor parallelism (TP), respectively. To use pure data parallelism, you should set `ICI_MESH_SHAPE` to `[NUM_GPUS, 1, 1]`.
-- `DCN_MESH_SHAPE`: This refers to the parallelism strategy for machines connected by a datacenter network. This is the generally parallel strategy used _across_ nodes.
+- `ICI_MESH_SHAPE`: This refers to the parallelism strategy used on chips connected by a fast network (e.g. NVLink). `ICI_MESH_SHAPE` typically has 3 dimensions, `[data, fsdp, tensor]`, corresponding to data parallelism (DP), fully-sharded data parallelism (FSDP/ZeRO-3), and tensor parallelism (TP), respectively. For example,to use pure data parallelism, you should set `ICI_MESH_SHAPE` to `[NUM_GPUS, 1, 1]`.
+- `DCN_MESH_SHAPE`: This refers to the parallelism strategy for machines connected by a datacenter network. In our case, this refers to the parallel strategy used _across_ nodes. It has the same dimensions as `ICI_MESH_SHAPE`.
```

Comment:
Does this incorporate the feedback from Azure team?

Reply:
No, this section incorporates their feedback: [https://github.com/NVIDIA/JAX-Toolbox/pull/345/files#diff-4183fc5677bb88511cd143f8177a00f7ef6752bb21ea0e98f96aa62781a22147R180](https://github.com/NVIDIA/JAX-Toolbox/pull/345/files#diff-4183fc5677bb88511cd143f8177a00f7ef6752bb21ea0e98f96aa62781a22147R180)